<?xml version="1.0"?>
<launch>


    <!-- load yaml file with all ros param -->
    <rosparam file="$(find vg_gpn)/config/vg_gpn_real.yaml" />



    <node name="vg_gpn_real" pkg="vg_gpn" type="vg_gpn_real.py"  output="screen"  >
    </node> 



<!-- 
    <node name="vg_sync_node" pkg="vg_gpn" type="vg_sync_node.py"  output="screen"  >
            <remap from="odom" to="/t265/odom/sample" />
            <remap from="vlp_pcl" to="/velodyne_points" />
            <remap from="cam_pcl" to="/d400/depth/color/points" />
            <remap from="cam_img" to="/d400/color/image_raw" />
    </node>  -->




</launch>

